src/lib/openapi-validation.ts

```diff
@@ -0,0 +1,83 @@
+import { z } from 'zod'
+
+export const parameterSchema = z.object({
```

Comment:
Let's use PascalCase for the zod schemas. I think it makes sense for this project since the schema function like classes with a factory function (`parse`).

```suggestion
export const ParameterSchema = z.object({
```